sand/extensions/search/wikidata_search.py

```diff
@@ -52,13 +63,28 @@ def get_props_search_params(self, search_text: str) -> Dict:
 
     def find_class_by_name(self, search_text: str) -> List[SearchResult]:
         """
-        Uses Wikidata API to search for classes using their name/text.
-        Uses local ID based class search to fetch label and description data.
+        Uses Wikidata API and local default classes to search for classes using their name/text.
         """
         request_params = self.get_class_search_params(search_text)
         api_data = requests.get(self.wikidata_url, request_params)
         search_results = api_data.json()['query']['search']
         payload_results = []
+
+        # search local classes
```

Comment:
it's better to implement this with composition pattern IMO

Reply:
Implemented composition pattern